### PR TITLE
ztp: set AMQP as default transport type

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jackding@gmail.com>

### CODE REVIEW

File: ztp/source-crs/PtpOperatorConfigForEvent.yaml [hunk 1]
```diff
@@ -10,5 +10,6 @@ spec:
     node-role.kubernetes.io/$mcp: ""
   ptpEventConfig:
     enableEventPublisher: true
-    # for AMQP Transport set transportHost to "amqp://amq-router.amq-router.svc.cluster.local"
-    transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
+    # for HTTP Transport set transportHost to
+    # "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
```
Comment (aneeshkp) [inline ztp/source-crs/PtpOperatorConfigForEvent.yaml:14]: remove hw-event , ptp don't use this

File: ztp/source-crs/HardwareEvent.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ metadata:
     ran.openshift.io/ztp-deploy-wave: "10"
 spec:
   nodeSelector: {}
-  # for AMQP Transport set transportHost to "amqp://amq-router.amq-router.svc.cluster.local"
-  transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
+  # for HTTP Transport set transportHost to
+  # "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
```
Comment (aneeshkp) [inline ztp/source-crs/HardwareEvent.yaml:11]: remove hw-event comment ptp doesn't allow hw-event pulisher

**Review by aneeshkp — APPROVED** [review]

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**